packages/core/modules-sdk/src/medusa-app.ts

```diff
@@ -85,6 +85,16 @@ export async function loadModules(
       let declaration: any = {}
       let definition: Partial<ModuleDefinition> | undefined = undefined
 
+      if (!isObject(mod) && typeof mod !== "boolean") {
+        throw new Error(
+          `Invalid module configuration for ${moduleName}. Expected boolean or a module configuration object, got ${typeof mod}. Check your medusa-config file.`
+        )
+      }
```

Comment:
suggestion: I dont think we should mention the medusa-config file here as it can be used as standalone to bootstrap modules in a lambda function fo example

Reply:
agree - good point